inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -241,6 +241,18 @@ dark_pink <- "#B45D75"
 theme_set(theme_episoap())
 ```
 
+### System dependencies
+
+You may need to install system dependencies to be able to generate this report:
+
+```sh
+# macOS
+brew install libsodium cmake
+
+# Linux (Debian based)
+apt install libsodium-dev cmake
+```
+
```

Comment:
````suggestion
<!--
### System dependencies

You may need to install system dependencies to be able to generate this report:

```sh
# macOS
brew install libsodium cmake

# Linux (Debian based)
apt install libsodium-dev cmake
```
-->

````

Can we make this a comment? It relates to previous discussion with @CarmenTamayo, where we decided to clarify who we are talking to in these templates:

- outside of code chunks: to the final reader of the report (decision makers or general public)
- as comments (inside or outside of code chunks): to the package users / the authors of the report